landoapi/models/landing_job.py

```diff
@@ -179,12 +183,15 @@ def landing_job_identifier(self) -> str:
         head = self.revisions[-1]
         if head.revision_id:
             # Return the Phabricator identifier if the head revision has one.
-            return f"D{self.revisions[-1].revision_id}"
+            return f"D{head.revision_id}"
+
+        # If there is no Phabricator identifier, return the first line of the
+        # non-try-syntax commit's message for the patch.
+        if len(self.revisions) > 1:
+            head = self.revisions[-2]
```

Comment:
Is there a more rigorous way of detecting a try syntax commit?

Reply:
There are quite a few different incantations of try commit syntax, so it isn't quite that simple. For the MVP I think this is a simple enough solution, and in the future we could add a check for specific types of try syntax. The key thing here is to just have some string that helps developers identify which patch this landing job is associated with.